omnisharp.el

```diff
@@ -1078,6 +1078,47 @@ cursor at that location"
         imenu-list)
     (error nil)))
 
+;; http://xahlee.blogspot.ru/2011/09/emacs-lisp-function-to-trim-string.html
+(defun trim-string (string)
```

Comment:
Let's remove this and use this instead:
https://github.com/magnars/s.el#s-trim-s